### PR TITLE
changing text class and its testcases

### PR DESCRIPTION
**Pull request by aliriaz186**

### CODE REVIEW

File: lib/strings/text.php [hunk 1]
```diff
@@ -14,36 +14,36 @@ public function __construct(string $str)
 
     public function isEmpty():bool
     {
-        if (!empty($this->str))
+        if (empty($this->str))
```
Comment (jsyspk) [inline lib/strings/text.php]: how about a one liner 

`return empty(this->str)?true:false;`

File: lib/strings/text.php [hunk 2]
```diff
@@ -14,36 +14,36 @@ public function __construct(string $str)
...
 
     public function length():int
     {
         return mb_strlen($this->str);
     }
+
     public function hasSpace():bool
     {
-        if (!strpos($this->str," "))
+        if (strpos($this->str," "))
```
Comment (jsyspk) [inline lib/strings/text.php]: go with one liner

File: tests/unit/strings/text-test.php [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+use PHPUnit\Framework\TestCase;
+
+class TextTest extends TestCase
+{
+    public function setUp(): void
+    {
+        parent::setUp(); // TODO: Change the autogenerated stub
+    }
+
+    public function test_must_accept_the_empty_string()
```
Comment (jsyspk) [inline tests/unit/strings/text-test.php]: `test_must_accept_the_empty_string`

we are not talking about validation or acceptance here

**Review by jsyspk — CHANGES_REQUESTED** [review]

File: lib/strings/text.php (hunk 1 above)
Comment (aliriaz186) [reply lib/strings/text.php]: resolved

File: lib/strings/text.php (hunk 2 above)
Comment (aliriaz186) [reply lib/strings/text.php]: resolved

**Pull request merged**